balancer/grpclb/grpclb_test.go

```diff
@@ -36,11 +36,13 @@ import (
 
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/balancer"
-	grpclbstate "google.golang.org/grpc/balancer/grpclb/state"
```

Comment:
This was being renamed because `state` is such a common name to want to use as a variable, and it's also not descriptive enough at the call sites ("`state.Set`: what's that?").  Can we change this back?  Or rename the package itself to `grpclbstate`?